src/mm-progress-bar/mm-progress-bar.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	scope.ProgressBar = Polymer({
+		is: 'mm-progress-bar',
+
+		behaviors: [
+			StrandTraits.Stylable
+		],
+
+		properties: {
+			ver: {
+				type: String,
+				value: '<<version>>'
+			},
+			backgroundColor: {
+				type: String,
+				value: '#131313'
```

Comment:
for the time being until its resolved how to do this in the future we have a bunch of preset values available on the Colors javascript object (see icon for example)